acceptance-tests/pom.xml

```diff
@@ -131,16 +191,99 @@
 
     <profiles>
         <profile>
-            <!-- Jenkins Acceptance Test Harness doesn't support Windows -->
-            <id>no-ath-on-win</id>
+            <id>acceptance-tests</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <includes>
+                                <include>it/**/*.java, !*SmokeTest*.java</include>
```

Comment:
I'd expect the `!` case here to be identical to the `include` case down below in the `smoke-tests` profile. Any reason why we can't do that? 

Between the two, I think I prefer the very explicit `!it/**/*SmokeTest.java` - it tells you exactly where to put a new smoke test instead of making you think you can just put `SmokeTest` anywhere in the class name.

I also think you should swap the `execution` and `configuration` sections here... just so the `acceptance-tests` profile failsafe block is as close as possible to the `smoke-tests` profile block.

Reply:
yep, I thought exclusion wouldn't work (I was getting weird result when I tried a mix of <includes> and <excludes>), but it's working now, so I must have done something wrong back then. :+1: